# Week 2: EKF Parameters
ekf_prediction_node:
  ros__parameters:
    process_noise_q: 0.01
    dt: 0.005

ekf_fusion_node:
  ros__parameters:
    process_noise_q: 0.01
    measurement_noise_r: 0.1
    dt: 0.005
